src/protocols/omron/conn.h

```diff
@@ -142,6 +142,10 @@ struct omron_request_t {
     /* used by the background thread for incrementally getting data */
     int request_size; /* total bytes, not just data */
     int request_capacity;
+    int response_size; /* size of data we expect to be returned by this request */
```

Comment:
IIRC, the request_size includes the EIP and CPF header in the overhead.   This should probably be changed so that these requests do not include that data.  It is extremely redundant.

Reply:
That's on me, not you!